nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts

```diff
@@ -1069,6 +1069,16 @@ export class FlowEffects {
             this.actions$.pipe(
                 ofType(FlowActions.openEditProcessorDialog),
                 map((action) => action.request),
+                switchMap((request) =>
+                    from(this.flowService.getProcessor(request.entity.id)).pipe(
+                        map((entity) => {
+                            return {
+                                ...request,
+                                entity
+                            };
+                        })
+                    )
```

Comment:
Do we need to consider the possibility of this endpoint failing?
When I force an error in the service and try to edit a processor config, nothing happens. Should we pop up a snackbar in this rare case? Or is ignoring it sufficient?

Reply:
I'm fine with not addressing it here as long it is accounted for.